src/app/routes/assets/verification-method/verification.service.spec.ts

```diff
@@ -16,7 +16,7 @@ describe('VerificationService', () => {
       providers: [
         { provide: IamService, useValue: iamServiceSpy },
         { provide: LoadingService, useValue: loadingServiceSpy },
-        { provide: ToastrService, useValue: toastrSpy }
+        { provide: SwitchboardToasterService, useValue: toastrSpy }
```

Comment:
I think you should also adjust toastrSpy, because it is right now `const toastrSpy = jasmine.createSpyObj('ToastrService', ['success']);`